homeassistant/components/vicare/binary_sensor.py

```diff
@@ -196,13 +195,7 @@ def __init__(
         self._api = api
         self.entity_description = description
         self._device_config = device_config
-        self._attr_device_info = DeviceInfo(
-            identifiers={(DOMAIN, device_config.getConfig().serial)},
-            name=device_config.getModel(),
-            manufacturer="Viessmann",
-            model=device_config.getModel(),
-            configuration_url="https://developer.viessmann.com/",
-        )
+        ViCareEntity.__init__(self, device_config)
```

Comment:
you can move this to the top of the constructor and change it to 
```suggestion
        super().__init__(device_config)
```

Reply:
You can do this with the rest as well

Reply:
This will work with multiple base classes?